bundles/org.openhab.binding.gree/src/main/resources/ESH-INF/i18n/gree.properties

```diff
@@ -0,0 +1,64 @@
+# GREE Binding
```

Comment:
Actually you don't need to add english "translations" to this file, which are already contained in the XML files.

Reply:
yes, I need to change the xml into @text/...
I learned it's helpful to have a English properties file, this let other people do translation

Reply:
done